solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -65,6 +69,8 @@ describe("RandomBeacon - Relay", () => {
   let signers: Operator[]
   let signersIDs: OperatorID[]
   let signersAddresses: Address[]
+  let tattletale: SignerWithAddress
```

Comment:
I know it's a v1 concept but maybe just `notifier`? This will be more consistent with what we already have in `notify*/report*` methods.

Reply:
Changed